### ISSUE TITLE
Dropdowns have broken ARIA  labelledby

### ISSUE THREAD

**Issue by silenceway** [zurb/foundation-sites#11572]
## What should happen?

aria-labelledby must point to label in dropdowns.

## What happens instead?

It's creating labelledby attribute but without point to the label.

## Possible Solution

Add id to this.$currentAnchor when created with GetYoDigits function.

## Test Case and/or Steps to Reproduce (for bugs)
Go to https://foundation.zurb.com/sites/docs/dropdown.html
Inspect dropdowns, they have aria-labelledby attribute but buttons doesn't have this id.

## Your Environment
Affects current foundation version 6.5

## Checklist
- [x] I have read and follow the CONTRIBUTING.md document.
- [x] There are no other issues similar to this one.
- [x] The issue title and template are correctly filled.
